thirdeye/thirdeye-frontend/tests/acceptance/edit-alert-test.js

```diff
@@ -42,5 +38,29 @@ test('visiting /manage/alerts/edit and checking that fields render correctly', a
       find(STATUS).get(0).innerText,
       'Active',
       'alert status is correct');
+
+    fillIn(ALERT_NAME, 'test_function_2');
+    click(STATUS_TOGGLER);
+    click(SUBMIT_BUTTON);
+  });
+
+  andThen(() => {
+    assert.equal(
+      currentURL(),
+      '/manage/alerts',
+      'correctly redirects to manage alerts page after edit'
+    );
+    visit(`/manage/alerts/${alert.id}`);
```

Comment:
nit: `visit` is an async helper. I believe the best practice for those helpers (visit, fillin, click, etc... ) is to have them as siblings of `andThen`s

https://guides.emberjs.com/v2.17.0/testing/acceptance/#toc_asynchronous-helpers

Reply:
Fixed.